### PR TITLE
ER-1008 - Searching by ProviderUkprn

### PR DESCRIPTION
**Pull request by chrisjensenuk**
Allowing searching and sorting by the training provider's UKPRN

### CODE REVIEW

File: SFA.DAS.VacancyServices.Search/SFA.DAS.VacancyServices.Search/Requests/VacancySearchSortType.cs [hunk 1]
```diff
@@ -6,6 +6,7 @@ public enum VacancySearchSortType
         Distance,
         ClosingDate,
         RecentlyAdded,
-        ExpectedStartDate
+        ExpectedStartDate,
+        ProviderUkprn
```
Comment (dev4valtech) [inline SFA.DAS.VacancyServices.Search/SFA.DAS.VacancyServices.Search/Requests/VacancySearchSortType.cs]: Not sure why would anybody want to sort by ukprn?

Comment (chrisjensenuk) [reply SFA.DAS.VacancyServices.Search/SFA.DAS.VacancyServices.Search/Requests/VacancySearchSortType.cs]: Thanks. The ticket says "The new UKPRN parameter should be optional and can be applied as an additional filter to existing searches and sorting logic.". I'll clarify what this actually means.

Comment (dev4valtech) [reply SFA.DAS.VacancyServices.Search/SFA.DAS.VacancyServices.Search/Requests/VacancySearchSortType.cs]: I have approved changes on that basis.

Comment (chrisjensenuk) [reply SFA.DAS.VacancyServices.Search/SFA.DAS.VacancyServices.Search/Requests/VacancySearchSortType.cs]: @dev4valtech  There is no requirement for sorting by UKPRN so it is just filtering. I'll update

**Pull request merged**